### ISSUE TITLE
No Tests

### ISSUE THREAD

**Issue by khanal-abhi** [dturrina/votesmanager#17]
I noticed that there are no tests. Were you considering adding tests as well?

**Comment by dturrina** [comment]
Yes sure. Any help is appreciated, so if you feel like writing tests, I'd be pleased to accept them :smiley:
